abi-compat/suppressions.abignore

```diff
@@ -7,3 +7,8 @@
 [suppress_variable]
   name = global_state
   type_name = NetplanState
+
+[suppress_type]
+  type_kind = typedef
+  name = NetplanDefType
```

Comment:
I agree we can allow this ABI change to the NetplanDeFType enum, as we're only changing the internal `NETPLAN_DEF_TYPE_MAX_` value, which does not affect existing consumers.